cloudberry/zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/impl/DataSetInfo.scala

```diff
@@ -108,6 +108,8 @@ object DataSetInfo {
           JsSuccess(StringField(name, isOptional))
         case DataType.Time =>
           JsSuccess(TimeField(name, isOptional))
+        case DataType.Json =>
+          JsSuccess(JsonField(name, isOptional))
```

Comment:
Hierarchy and Bag fields are stored as Json format in MySQL. So I created JsonField and DataType.Json.